tests/integration-tests/tests/iam/test_iam.py

```diff
@@ -114,7 +114,7 @@ def test_iam_policies(region, scheduler, pcluster_config_reader, clusters_factor
 
 def _test_s3_access(remote_command_executor, region):
     logging.info("Testing S3 Access")
-    result = remote_command_executor.run_remote_command(f"AWS_DEFAULT_REGION={region} sudo aws s3 ls").stdout
+    result = remote_command_executor.run_remote_command(f"AWS_DEFAULT_REGION={region} aws s3 ls").stdout
```

Comment:
how come these changes to integ tests?

Reply:
They were in your patch, I have put them in a separate commit.

Reply:
hum I might have added these changes by mistake when rebasing.